_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,361 @@
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 1.1.2][hwi 1.1.2]: this release includes support
+  for handling [PSBTs][topic psbt] with serialized previous segwit
+  transactions, which is now required or suggested for several hardware
+  wallets in response to the [fee overpayment attack][news101 fee
+  overpayment].
+
+- [LND 0.10.2-beta.rc2][lnd 0.10.2-beta]: this release candidate for an
+  LND maintenance release is now available for testing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19305][] adds release notes for the upcoming codebase
+  transition from the C++11 standard to C++17. The plan is to add C++17
+  compatibility in the 0.21 release ([expected][Bitcoin Core #18947] end
+  of 2020) and break C++11 compatibility later in 0.22 (expected mid-2021).
+  Although this change won't affect most users, those planning to build
+  the 0.22 release on older systems may want to check their toolchain's standard
+  support against the [C++ compiler support matrix][cpp compiler support].
+
+- [Bitcoin Core #11413][] updates the `bumpfee`, `fundrawtransaction`,
+  `sendmany`, `sendtoaddress`, and `walletcreatefundedpsbt` RPCs to
+  allow manually specifying the feerate to use in the newly created or
+  updated transaction.  By default, these commands still use either an
+  automatic feerate computed by Bitcoin Core's built-in transaction fee
+  estimate or the fallback feerate if there's not enough data for
+  estimation.  For details on how to use the updated RPCs, see the
+  [proposed release note][bcc11413rn].  Despite not affecting any major
+  system, this PR was open for almost three years---the second longest
+  of any PR that has been merged into Bitcoin Core so far.  We thank the
+  author, Kalle Alm, and everyone else involved for their persistence.
+
+- [Eclair #1466][] decreases the length of time that an Eclair node will wait
+  before closing a channel when a peer becomings unresponsive while an HTLC is
+  pending. To prevent payments from becoming stuck forever, each HTLC includes
+  a timelock. If the timelock expires, then the sending party can reclaim
+  the funds in the HTLC. To prevent the sending party from pulling back
+  funds that an intermediate party has already paid to the next hop, the
+  intermediate party must settle the pending HTLC onchain if the sending
+  party becomes unresponsive. This PR increases Eclair's safety window---it
+  will now broadcast the on chain transaction 24 blocks before the HTLC
```

Comment:
onchain

Reply:
perhaps end the sentence at "6" or "instead of the previous value of six."

Should we mention something along the lines that this change effectively lowers `expiry-delta-blocks`, so Eclair users may wish to increase it.

Reply:
I think we'll skip mentioning that to avoid having to add extra information about the relationship between the parameters.  Hopefully Eclair will put something about that into their release notes when they release this change.